### PR TITLE
Add more Chrome & Firefox version numbers for tabs webextensions API

### PR DESCRIPTION
**Pull request by gregorypappas**
#### Summary

A few apis are missing correct values for the `version_added` field.

#### Test results and supporting details

https://web.archive.org/web/20090918121259/http://code.google.com/chrome/extensions/tabs.html
https://web.archive.org/web/20160701175838if_/https://developer.chrome.com/extensions/tabs#type-MutedInfo
https://web.archive.org/web/20160701175838if_/https://developer.chrome.com/extensions/tabs#properties
https://web.archive.org/web/20160701175838if_/https://developer.chrome.com/extensions/tabs#property-Tab-active
https://web.archive.org/web/20160701175838if_/https://developer.chrome.com/extensions/tabs#property-Tab-highlighted
https://web.archive.org/web/20160701175838if_/https://developer.chrome.com/extensions/tabs#method-duplicate
https://web.archive.org/web/20180706052022/https://developer.chrome.com/extensions/whats_new
In this snapshot from Nov 22, 2011, there is no incognito property: https://web.archive.org/web/20111122231840/http://code.google.com/chrome/extensions/tabs.html
In this snapshot from Nov 30, 2011, the incognito property is present: https://web.archive.org/web/20111130085758/http://code.google.com/chrome/extensions/tabs.html
So I reason that the property was added in version 16 (stable release 2011-12-13).
sessionId in Firefox: https://searchfox.org/mozilla-central/rev/b303437fafc9d962e93f2322711355da25ed2a26/browser/components/extensions/ext-utils.js#708

### CODE REVIEW

**Comment by gregorypappas** [comment]
Hi @rebloor,
Would you be willing to review this? Thanks!

File: webextensions/api/tabs.json [hunk 1]
```diff
@@ -1001,6 +1001,9 @@
                   "version_added": "15"
                 }
               }
+            },
+            "status": {
+              "deprecated": true
```
Comment (queengooborg) [inline webextensions/api/tabs.json:1006]: WebExtensions doesn't actually accept `status` blocks as they're not standard:

```suggestion
```

**Comment by rebloor** [comment]
@gregorypappas have you finished your changes?

**Comment by gregorypappas** [comment]
@rebloor Yes, I am done making changes.

**Comment by rebloor** [comment]
Thanks @gregorypappas, looks good.

**Pull request merged**